README.md

```
@@ -1,52 +1,98 @@
-- **🔍 Deep Clean Removal**: Stages every candidate directory before execution (`~/.config`, `~/.cache`, `~/.local/share`, `/etc`, `/var/log`).
-- **📊 1-Click Interactive Stat Cards**: Jump between `All Apps`, `Flatpak`, `Snap`, `Tools`, `Cache`, and `Reclaimable` with keys <kbd>1</kbd>–<kbd>6</kbd> or <kbd>[</kbd> / <kbd>]</kbd>.
-- **⚡ Instant Bloat Sorting (<kbd>o</kbd>)**: Cycle sorting order (`Size ↓`, `Name A-Z`, `Source`, `Date ↓`) to find massive disk-hogging applications immediately.
-- **💾 Real-Time Disk Gauge**: Live disk utilization monitoring displayed in the header.
-- **🔒 Sudo Batching & Safety First**: Home-directory paths are removed as the current user. System paths are batched into **one single sudo prompt** per uninstall. Core system packages (`glibc`, `linux`, `systemd`, current shell) are protected and refused by default.
-- **📄 Audit Trail & JSON Export**: Every removal is logged to `~/.local/share/veet/history.log`. Press <kbd>e</kbd> to export an inventory report to `apps-report.json`.
+## Core Capabilities
+
+<table>
+  <tr>
+    <td width="50%" valign="top">
+      <h4><img src="assets/icons/sources.svg" width="18" height="18" align="center" /> Multi-Source Parallel Detection</h4>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)